src/main/java/org/generationcp/middleware/api/germplasm/GermplasmServiceImpl.java

```diff
@@ -87,6 +94,9 @@
 		RECURSIVE
 	}
 
+
+	public static final List<String> SYSTEM_NAMES =
```

Comment:
We have the same list of system names in `GermplasmNameTypeValidator` in BMSAPI. We should just maintain one list